### ISSUE TITLE
Species-based earthtones

### ISSUE THREAD

**Issue by wcornwell** [wcornwell/earthtones#6]
![image](https://cloud.githubusercontent.com/assets/5534161/18495436/3de61314-7a61-11e6-9aaf-cec3580a4519.png)

**Comment by pssguy** [comment]
gotta admit tropical fish would give wider range than earthtones

**Comment by wcornwell** [comment]
definitely.  but it's a harder problem because of identifying what is fish and what is not-fish in the image

**Comment by mitchest** [comment]
We should see if we can use a Google API to extract images from street view or the under water ones!

**Comment by wcornwell** [comment]
yeah the streetview of the great barrier reef would be cool (and very blue) 

some discussion about streetview here: http://stackoverflow.com/questions/39197022/a-way-to-access-google-streetview-from-r?noredirect=1#comment66065688_39197022

**Comment by wcornwell** [comment]
@mitchest There is some very interesting stuff here: https://en.wikipedia.org/wiki/Edge_detection

Is this how you detect the edges of tree canopies?

**Comment by wcornwell** [comment]
also very interesting read here: https://colinpriest.com/tag/edge-detection/
